src/app/services/hero/hero.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+
+import { HeroModel } from 'models/animal/hero/hero.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class HeroService {
+  create(hero: HeroModel, isHurt: boolean) {
```

Comment:
Return type is missing

Reply:
I doubted this because it's no HeroModel. Or should **isHurt** be optional so I can have HeroModel return type?

Reply:
yes

Reply:
isHurt should be optional also